addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar.scss

```diff
@@ -51,12 +55,31 @@
 // Style
 // ------------------------------------------------------------------
 
-.o_ChatterTopbar .o-active {
-    color: $o-brand-odoo;
-    background-color: lighten(gray('300'), 7%);
-    border-color: gray('300');
-    border-bottom-color: transparent;
-    border-top-color: transparent;
+.o_ChatterTopbar_actions.o-has-active-button {
+    border-bottom-color: $border-color;
+}
+
+.o_ChatterTopbar_button {
+    border-radius: 0;
+
+    &:hover{
+        background-color: gray('300');
+    }
+
+    &.o-active {
+        color: $o-brand-odoo;
+        background-color: lighten(gray('300'), 7%);
+        border-left-color: $border-color;
+        border-right-color: $border-color;
+
+        &.o-bordered {
+            border-top-color: $border-color;
+        }
```

Comment:
same reasoning here, no way to have only one `border-color: $border-color;` for everything?

Reply:
Not really, you need specificity for color overridding and you need top border to be only here if o-bordered. You could maybe translate this logic in display by changing borders size to $border-size when need but it's the same at the end.